common/eslint-local-rules/__tests__/matchingTranslationKeys.spec.js

```diff
@@ -39,134 +41,133 @@ ruleTester.run('local-rules/matching-translation-keys', ruleInstance, {
       name: 'allows correct key in js',
       code: '$tc("components.hello.world")',
       options: options,
-      filename: '/app/src/components/hello.js',
+      filename: '/src/components/hello.js',
```

Comment:
I did not get the packageDirectory mock to work, so i used a simple solution.
Else it works in the docker container, but not when run locally with an interpreter.
ok @carlobeltrame ?

Reply:
As long as the tests work in CI and when run in the way we have it documented [in the wiki](https://github.com/ecamp/ecamp3/wiki/Testing-guide), fine by me